auth0/resource_auth0_organization.go

```diff
@@ -207,12 +210,14 @@ func updateOrganizationConnections(d *schema.ResourceData, api *management.Manag
 }
 
 func expandOrganization(d *schema.ResourceData) *management.Organization {
-	metadata := Map(d, "metadata")
-
 	organization := &management.Organization{
 		Name:        String(d, "name"),
 		DisplayName: String(d, "display_name"),
-		Metadata:    &metadata,
+	}
+
+	if d.HasChange("metadata") {
```

Comment:
While implementing the new resource and adding minimal tests for it, it was spotted that we need a HasChange over here when we first create the org, otherwise if we don't have the metadata the first time we create the org it will get sent as "null" and the API doesn't accept that.